mysql-test/main/repair_symlink-5543.test

```diff
@@ -12,6 +12,7 @@ insert t1 values (1);
 --echo # Some systems fail with errcode 40, or 90 (MIPS) when doing openat,
 --echo # while others don't have openat and fail with errcode 20.
 --replace_regex / '.*\/t1/ 'MYSQL_TMP_DIR\/t1/ /[49]0/20/ /".*"/"<errmsg>"/
+--replace_regex / '.*\/t1/ 'MYSQL_TMP_DIR\/t1/ /85/20/ /".*"/"<errmsg>"/
```

Comment:
I think we should still retain 40/20 and 90/20 replacement, and just add /85/20/ for AIX . Because comment above says that 40 and 90 were seen somewhere.

Reply:
Maybe I miss something, but the line 14 with 40/20 and 90/20 replacement is kept. Another is added for AIX.

Reply:
you're right,  I have overseen that , sorry. I suggest to add your 85/20 to the existing line
```
--replace_regex / '.*\/t1/ 'MYSQL_TMP_DIR\/t1/  /[49]0/20/  /85/20/  /".*"/"<errmsg>"/
```
like it is done elsewhere.

Reply:
Done (here and below).